app/client/jsx/Room.jsx

```diff
@@ -39,7 +39,8 @@ class Room extends Component {
            displayName: this.props.displayName,
            avatar: this.props.avatar,
            roomName: roomData.name,
-           height: roomData.videoHeight
+           height: roomData.videoHeight,
```

Comment:
Is this still used or do we rely on room-specific css for this?

Reply:
Also asking bc I'm wondering whether or not to put it in the db schema

Reply:
It's still used in jitsi rooms, looks to me.

Reply:
Or all rooms with Jitsi? It's in the Jitsi class, but it seems like video expands to the space?
```
            const options = {
                roomName: this.props.jitsiData.roomName,
                height: this.props.jitsiData.height || 500,
                parentNode: document.getElementById('jitsi-container'),
                interfaceConfigOverwrite: {
                    // filmStripOnly: true,
                    SHOW_JITSI_WATERMARK: false,
                    DEFAULT_REMOTE_DISPLAY_NAME: 'Fellow Clarendonite',
                    SHOW_WATERMARK_FOR_GUESTS: false,
                    TOOLBAR_BUTTONS: this.toolbarButtons,

                },
                configOverwrite: {
                    disableSimulcast: false
                }
```

Reply:
Yeah it's not used anymore, CSS is overriding

Reply:
Mmk, would you be able to get rid of those references in this PR? And I won't include it in the db models

Reply:
Sure